ideascube/serveradmin/catalog.py

```diff
@@ -176,6 +185,97 @@ def unregister_site(self, package):
         except FileNotFoundError as e:
             printerr(e)
 
+class MediaCenter(Handler):
+    def commit(self):
+        for pkg in self._removed:
+            self.remove_pkg(pkg)
```

Comment:
I've made like Nginx/StaticSite handler/package.

That is Package that has an `install` method, who physicaly install files on hd.
The `commit` handler's method update config/db.